flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/context/SessionContextTest.java

```diff
@@ -202,15 +240,32 @@ private SessionContext createSessionContext() throws Exception {
         return SessionContext.create(defaultContext, "test-session");
     }
 
-    private Map<String, String> getConfigurationMap(SessionContext context) {
-        return context.getExecutionContext()
+    private Map<String, String> getConfigurationMap() {
+        return sessionContext
+                .getExecutionContext()
                 .getTableEnvironment()
                 .getConfig()
                 .getConfiguration()
                 .toMap();
     }
 
-    private Configuration getConfiguration(SessionContext context) {
-        return context.getExecutionContext().getTableEnvironment().getConfig().getConfiguration();
+    private Configuration getConfiguration() {
+        return sessionContext
+                .getExecutionContext()
+                .getTableEnvironment()
+                .getConfig()
+                .getConfiguration();
+    }
+
+    private void validateAddJarException(String jarPath, String errorMessages) {
+        Set<URL> originDependencies = sessionContext.getDependencies();
+        try {
+            sessionContext.addJar(jarPath);
```

Comment:
Please add `fail("should fail");`, otherwise the test will pass if no exception is thrown.